packages/core/saved-objects/docs/openapi/entrypoint.yaml

```diff
@@ -42,6 +42,33 @@ paths:
 #    $ref: 'paths/api@saved_objects@{type}@{id}.yaml
 #  'api/saved_objects/resolve/{type}/{id}':
 #    $ref: 'paths/api@saved_objects@resolve@{type}@{id}.yaml'
+  '/api/encrypted_saved_objects/_rotate_key':
+    $ref: 'paths/api@encrypted_saved_objects@_rotate_key.yaml'
+# '/api/saved_objects/_export':
+# $ref: 'paths/api@saved_objects@_export.yaml'
+# '/api/saved_objects/_import':
+# $ref: 'paths/api@saved_objects@_import.yaml'
+# '/api/saved_objects/_resolve_import_errors':
+# $ref: 'paths/api@saved_objects@_resolve_import_errors.yaml'
```

Comment:
Looking at the `bundle.{json|yaml}`, shouldn't these be uncommented? 🤷

Reply:
This is odd, when I look at my local they are uncommented and when I tried to pull this down they appear to be uncommented.

Reply:
@amyjtechwriter  I pushed [`1391fd8` (#162566)](https://github.com/elastic/kibana/pull/162566/commits/1391fd804180bb3feb1729c5748918d45afb3281) to uncomment the lines. Can you review the https://github.com/elastic/kibana/pull/162566/files to let me know this looks as intended? Thanks!

Reply:
The entrypoint file LGTM now, thanks!